modules/price/src/Price.php

```diff
@@ -247,6 +247,22 @@ public function lessThanOrEqual(Price $price) {
   }
 
   /**
+   * Rounds the price to the specified precision.
+   *
+   * @param string $precision
+   *   The precision.
+   * @param int $mode
+   *   The rounding mode.
+   *
+   * @return static
+   *   The resulting price.
+   */
+  public function round($precision = '2', $mode = PHP_ROUND_HALF_UP) {
```

Comment:
Also, we have the currency code but no data. Should we keep `$precision` and force invokers to get the currency's decimal count, or find some way to make the price object aware of it.